src/resolver.ts

```diff
@@ -273,6 +273,10 @@ export class Resolver extends DiagnosticEmitter {
           case CommonSymbols.native: return this.resolveBuiltinNativeType(node, ctxElement, ctxTypes, reportMode);
           case CommonSymbols.indexof: return this.resolveBuiltinIndexofType(node, ctxElement, ctxTypes, reportMode);
           case CommonSymbols.valueof: return this.resolveBuiltinValueofType(node, ctxElement, ctxTypes, reportMode);
+          case CommonSymbols.ReturnType:
+          case CommonSymbols.returnof: {
+            return this.resolveBuiltinReturnTypeType(node, ctxElement, ctxTypes, reportMode);
```

Comment:
nit / style: no braces like the cases above

Reply:
Fixed.

It was two symbols, so I wasn't sure if this was the right way to express this. Thanks for the feedback.